### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by xiortgav** [PrismarineJS/mineflayer#1913] [Q&A] [answered]
Error: connect ETIMEDOUT ip
at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16) {
errno: -110,
code: 'ETIMEDOUT',
syscall: 'connect',
address: 'ip',
port: 25565
}

**Comment by amoraschi** [comment, marked as answer]
EDIT: That means the server is rejecting the connection.
